README.md

```diff
@@ -946,8 +946,46 @@ constructor(props) {
 # Conditional rendering
 
 ### AngularJS
+Angularjs 1.x has three ways to perform conditional rendering:   `ng-if`, `ng-switch` and `ng-hide/ng-show`.
+ 
+ 
+```js
```

Comment:
Could you fix the formatting?
- new line after `AngularJS`
- remove extra second line above
- remove double space before `ng-if`
- space before `"Registered"`